### DISCUSSION TITLE
problema

### DISCUSSION THREAD

**Discussion by Dari0GR** [Homebrew/discussions#3217] [Getting started]
Cuando inicio terminal me sale lo siguiente:
/Usuarios/dariogr/.zprofile:1: permiso denegado: /opt/homebrew/bin/brew

Alguien podría ayudarme por favor?

**Comment by SMillerDev** [comment]
What does `ls -al /opt/homebrew/bin/` say?

**Reply by Dari0GR** [reply]
![Captura de Pantalla 2022-04-27 a la(s) 9 03 19](https://user-images.githubusercontent.com/102642131/165537012-1bb4fcc9-9355-4a4f-af6f-3f1f9bf14210.png)

**Reply by Dari0GR** [reply]
![Captura de Pantalla 2022-04-27 a la(s) 9 06 01](https://user-images.githubusercontent.com/102642131/165537452-f877e1a5-1b9c-4a3e-ac9e-b3838b619950.png)

**Reply by SMillerDev** [reply]
It's easier if you just copy the text here instead of screenshots since that's easier to read.

---
It seems like your install of Homebrew did not work correctly. You can [uninstall Homebrew](https://github.com/homebrew/install#uninstall-homebrew) and run the installer again after to try and fix this.

**Reply by Dari0GR** [reply]
Volvi a instalar y ya me funciono sin problema, no se que fue. Te agradezco!!
